doc/portfile.7

```diff
@@ -102,42 +104,57 @@ decreased.
 .Sy Example:
 .Dl epoch 1
 .It Ic description
-One line description of the software and what it does.  To appear in the
-description, brackets and semi-colons need to be escaped with a backslash (i.e.
-they must be written as "\\[", "\\]" and "\\;").  The escape sequences listed in
+One-line description of the software and what it does.
+To appear in the description, brackets and semi-colons need to be escaped with
+a backslash (i.e., they must be written as
+.Ql \e[ ,
+.Ql \e]
+and
+.Ql \e; ) .
+The escape sequences listed in
 .Xr re_syntax n
-may also be used, with the exception of \\n, \\r and \\f.
+may also be used, with the exception of
+.Ql \en ,
+.Ql \er
+and
+.Ql \ef .
 .br
 .Sy Type:
 .Em required
 .br
 .Sy Example:
-.Dl description Dictionary Server Protocol (RFC2229) client
+.Dl description IMAP email checker
 .It Ic long_description
-A verbose description of the software and what it does.  To appear in the
-description, brackets and semi-colons need to be escaped with a backslash (i.e.
-they must be written as "\\[", "\\]" and "\\;").  The escape sequences listed in
+A verbose description of the software and what it does.
+To appear in the description, brackets and semi-colons need to be escaped with
+a backslash (i.e., they must be written as
+.Ql \e[ ,
+.Ql \e]
+and
+.Ql \e;) .
```

Comment:
This renders as
```
(i.e., they must be written as `\[', `\]' and `\;)'
```
Shouldn't the closing `)` come after the quotation mark?

Reply:
Yep.  A typo, now fixed.